site/content/en/docs/Installation/Install Agones/yaml.md

```diff
@@ -36,6 +38,19 @@ Note: `pull` command was introduced in Helm version 3.
 
 You can also find the install.yaml in the latest `agones-install` zip from the [releases](https://github.com/googleforgames/agones/releases) archive.
 
+### Uninstalling Agones
+
+To uninstall/delete the `Agones` deployment and delete `agones-system` namespace:
+
+```bash
+$ kubectl delete fleets --all --all-namespaces
+$ kubectl delete gameservers --all --all-namespaces
+$ kubectl delete -f https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch >}}/install/yaml/install.yaml
+$ kubectl delete namespace agones-system
+```
+
+Note: you should wait up to a couple of minutes until all resources described in `install.yaml` file would be deleted.
```

Comment:
Not sure if we want to add any info for troubleshooting if you end up with the namespace in a `terminating` state, WDYT?

Reply:
I can not reproduce an original issue if I follow these 4 commands one by one. If you can, please tell me so I can add some info into a Troubleshooting section.
I have added this `Note`, because `kubectl delete` is frozen up to 2 minutes before exiting. Also I saw a similar comment in a separate ticket about this being slow.

Reply:
You are right if you follow those commands it doesnt occour :+1:  was meaning if anyone didnt uninstall correctly it would be helpful.

Reply:
@domgreen maybe a separate PR in https://agones.dev/site/docs/guides/troubleshooting/ ?